src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -390,6 +390,15 @@
     }
   }
 
+  @media (prefers-reduced-motion: reduce) {
```

Comment:
can this be moved inside of `.eyb-sheet` ?

Reply:
Will try moving that inside of `.eyb-sheet`

Reply:
That did not seem to cause issue either. Pushed a commit with both changes.